Sources/AWSLambdaRuntimeCore/LambdaHandler.swift

```diff
@@ -163,6 +191,20 @@ extension ByteBufferLambdaHandler {
     }
 }
 
+extension ByteBufferLambdaHandler {
+    /// Initializes and runs the lambda function.
+    ///
+    /// If you precede your ``ByteBufferLambdaHandler`` conformer's declaration with the
+    /// [@main](https://docs.swift.org/swift-book/ReferenceManual/Attributes.html#ID626)
+    /// attribute, the system calls the conformer's `main()` method to launch the lambda function.
+    ///
+    /// The lambda runtime provides a default implementation of the method that manages the launch
+    /// process.
+    public static func main() {
+        _ = Lambda.run(configuration: .init(), handlerType: Self.self)
```

Comment:
should we mark Lambda::run @discardableResult?

Reply:
I think the `Lambda.run` method should not return anything. But I haven't made my mind up about this fully. Would like to tackle later.